api/users/views.py

```diff
@@ -18,11 +21,13 @@ class UserMixin(object):
 
     def get_user(self, check_permissions=True):
         obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
-        if check_permissions:
+
+        if check_permissions & obj.is_disabled is False:
```

Comment:
So are we sure that is_active is definitely always correct now? It definitely sounds better than just checking is_disabled, but I'm very tempted to verify against the 6 or 7 different conditions under which a user might be incorrectly classified as not inactive and ensuring that none of them hit. I'm a little jaded about active users right now, as you can imagine.